core/src/repl/mod.rs

```diff
@@ -270,17 +272,16 @@ impl<EC: EvalCache> Repl for ReplImpl<EC> {
 
         let wildcards =
             typecheck::type_check(&term, self.env.type_ctxt.clone(), self.vm.import_resolver())?;
-        // Substitute the wildcard types for their inferred types
-        // We need to `traverse` the term, in case the type depends on inner terms that also contain wildcards
+        // Substitute the wildcard types for their inferred types We need to `traverse` the term, in
+        // case the type depends on inner terms that also contain wildcards
         let term = term
             .traverse(
-                &|rt: RichTerm, _| -> Result<RichTerm, std::convert::Infallible> {
+                &mut |rt: RichTerm| -> Result<RichTerm, Infallible> {
```

Comment:
I'm not necessarily convinced that importing a symbol for a single use is better than inlining it. Do you happen to know of any kind of official and widespread guideline telling to do so?

Reply:
I don't think there's any. That said, I tend to value type clarity over avoiding `use`s. It's not like it adds any real overhead, but long type names definitely make code that much harder to understand. So the cost calculation for me becomes:
- near zero cost to adding the use
- definitely non-zero cost to removing it

So I tend to add it. That said, not like this is complicated code; We could remove it if you prefer.

Reply:
I was mostly curious about the status quo elsewhere. I personally don't entirely agree (sometimes I find full qualified names clearer, especially when there are a lot of symbols floating around, because you immediately see what namespace they relate to without having to go to their introduction site). But `Infallible` is pretty standard, it's mostly a matter of taste, and above all it's quite unimportant, so please don't bother removing this :slightly_smiling_face: